de.js

```diff
@@ -920,63 +920,63 @@ chall.name.dmg {0} Schaden zufügen {1}
+modes.prophunt.limit [Prop Hunt] Du kannst dich nichtmehr verwandeln
+modes.prophunt.left [Prop Hunt] Du hast noch {0} Verwandlungen übrig
+modes.prophunt.rotation.on [Prop Hunt] Rotation ein
+modes.prophunt.rotation.off [Prop Hunt] Rotation aus
+modes.prophunt.rotation.reset [Prop Hunt] Rotation zurückgesetzt.
+commands.login Logge dich ein um diesen Befehl zu verwenden
+commands.connected {0} Verbunden, {1} im Menü
+commands.host {0} ist der momentanige Host.
+commands.noclip.on Noclip - Ein
+commands.noclip.off Noclip - Aus
+commands.god.on God mode - Ein
+commands.god.off God mode - Aus
+commands.god.on.other God mode - Ein für {0}
+commands.god.off.other God mode - Aus für {0}
+commands.smite.on Smite - Ein
+commands.smite.off Smite - Aus
+commands.smite.on.other Smite - Ein für {0}
+commands.smite.off.other Smite - Aus für {0}
+commands.perk {0} hat {1} erhalten
 commands.maxperk Gave {0} MAX {1}
-commands.unlimited.on Unlimited Ammo - Enabled
-commands.unlimited.off Unlimited Ammo - Disabled
-commands.unlimited.on.other Unlimited Ammo - Enabled for {0}
-commands.unlimited.off.other Unlimited Ammo - Disabled for {0}
-commands.spraybypass.on Unlimited Ammo - Enabled
-commands.spraybypass.off Unlimited Ammo - Disabled
-commands.spraybypass.on.other Unlimited Ammo - Enabled for {0}
-commands.spraybypass.off.other Unlimited Ammo - Disabled for {0}
-commands.suspect.invalid Invalid suspect info
-commands.suspect.menu Suspect in menu
-commands.suspect.left Suspect has left
+commands.unlimited.on Unlimited Ammo - Ein
+commands.unlimited.off Unlimited Ammo - Aus
+commands.unlimited.on.other Unlimited Ammo - Ein für {0}
+commands.unlimited.off.other Unlimited Ammo - Aus für {0}
+commands.spraybypass.on Unlimited Ammo - Ein
+commands.spraybypass.off Unlimited Ammo - Aus
+commands.spraybypass.on.other Unlimited Ammo - Ein für {0}
+commands.spraybypass.off.other Unlimited Ammo - Aus für {0}
+commands.suspect.invalid Ungültige Tatverdächtiger Info
+commands.suspect.menu Tatverdächtiger ist im Menü
+commands.suspect.left Tatverdächtiger hat das Spiel verlassen
 commands.suspect.found Suspect Found. Spectating: {0}, Spawned: {1}, Name: {2}
-commands.forfeit.next Must play atleast 1 round
-commands.forfeit.wait Wait till match starts to forfeit
-commands.forfeit.later Please try again in a few seconds
-commands.forfeit.voted You already voted.
-commands.forfeit.limit You can only start one vote per game
-commands.forfeit.ended Forfeiting Game
-commands.forfeit.ended.other Other Team Forfeited
-server.message.join {0} joined the game
-server.message.left {0} left the game
-server.message.kicked {0} was kicked
-server.message.banned {0} was banned
-server.message.votekicked {0} was vote kicked
-server.message.arrested {0} was arrested by the K.P.D
+commands.forfeit.next Du musst mindestens eine Runde spielen
+commands.forfeit.wait Warte bis das Match startet um aufzugeben
+commands.forfeit.later Versuche es in ein paar Sekunden erneut
```

Comment:
check with consistency on 820, nochmal/erneut mean the same thing, just choose one